docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/install-and-configure-stripe-prerequisites.md

```diff
@@ -217,6 +217,100 @@ use Spryker\Zed\OauthClient\Communication\Plugin\Payment\AccessTokenPaymentAutho
 
 ```
 
+## Using Stripe in Marketplace context
+
+Install the required modules:
+
+```bash
+composer require spryker/merchant-app spryker/merchant-app-merchant-portal-gui
+```
+
+
+### config_default.php
+
+Update your `config_default.php` file with the following changes:
```

Comment:
What does this configuration do?

Reply:
It will be used to to set headers to requests made from SCOS to ACP.